packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -312,9 +312,15 @@ export const Subscriptions = ({
 
       <div className="child-views" data-testid="subscription-management-loaded">
         <div className="settings-child-view support">
-          <div className="settings-unit">
+          <section
+            className="settings-unit"
+            aria-labelledby="subscriptions-support"
```

Comment:
Just curious, why make the inner tag a `section` and leave the outer as a `div` instead of swapping them + the label? (I didn't check into your branch FWIW, maybe that'd make it more obvious.) I like the rule of thumb that things are `sections` if they've got an `aria-labelledby` as you've done here - and it might even be nice to have that as a tip in `fxa-react`'s README even, which probably needs to be propagated out to other areas of FxA's front-end once more things are shared - though I know we've got section tags that are much more arbitrary throughout the codebase (which I think is fine).

Reply:
Yeah, that all seems good to me. I think at least in the past it was expected that `article`s contain a `header` and a `footer`, but I just looked it up to double check myself and that doesn't seem to be the case. I guess `article`s are meant to be isolated or self-contained pieces with at least an `h` tag while `section` is more arbitrary and up to devs to organize it how we want. I don't think we're using `article` anywhere actually except in some old content-server stuff. I'm not sure it makes any semantic difference anyway for screenreaders but I could be wrong there depending on the OS / screenreader.